platform/core/src/services/CustomizationService/CustomizationService.test.js

```diff
@@ -160,5 +165,149 @@ describe('CustomizationService.ts', () => {
       expect(result.label).toBe(testItem.label);
       expect(result.value).toBe(props.testAttribute);
     });
+
```

Comment:
Add a whole bunch of unit tests to show how the services can be used/how to handle the append etc.